docs/book.toml

```diff
@@ -3,4 +3,7 @@ authors = ["grjte"]
 language = "en"
 multilingual = false
 src = "src"
-title = "Polygon Miden's AirScript"
+title = "AirScript by Polygon Miden"
+
+[output.linkcheck]
+warning-policy = "ignore"
```

Comment:
With this line, I couldn't build the book locally. But once I deleted it, it worked fine. Maybe there are some broken links somewhere?